meganav.yaml

```diff
@@ -495,7 +495,7 @@ use_case:
           description: Smart displays and kiosks
           url: /internet-of-things/smart-displays
 
-  secondary_links:
+  highlighted_secondary_links:
```

Comment:
This is a temporary name, speaking with Juan to come up with a more permanent one